lib/widgets/emoji_reaction.dart

```diff
@@ -162,11 +162,9 @@ class ReactionChip extends StatelessWidget {
                     // ones (probably a good thing).
                     SizedBox(height: _squareEmojiScalerClamped(context).scale(_squareEmojiSize)),
                     Flexible( // [Flexible] to let text emojis expand if they can
-                      child: Padding(padding: const EdgeInsets.symmetric(horizontal: 3, vertical: 1),
+                      child: Padding(padding: const EdgeInsets.symmetric(horizontal: 3),
                         child: emoji)),
-                    // Added vertical: 1 to give some space when the label is
-                    // taller than the emoji (e.g. because it needs multiple lines)
```

Comment:
As Greg suggested:
      https://github.com/zulip/zulip-flutter/pull/442/commits/ec4a21fdccdb21774ca656082fa921161d6b1ae0#r1442492101

Link is broken — GitHub links like …/pull/NNN/commits/HHH/… are unstable, because GitHub breaks them if the commit is no longer in the latest revision of the PR. Instead link to discussions on the "Conversation" tab of the PR, like https://github.com/zulip/zulip-flutter/pull/442#discussion_r1442492101 .

Reply:
Ah indeed. I knew that, but missed it in this case; thanks for the catch. 🙂